scijava/scijava-ops/pom.xml

```diff
@@ -118,10 +118,6 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.8.1</version>
 				<configuration>
-					<annotationProcessors>
```

Comment:
Ah, nice! If we don't need these anymore, do we even need the maven-compiler-plugin block in this POM at all anymore?

Reply:
Short answer: [no](https://github.com/scijava/incubator/blob/07342c480541a31924d8f79ee247f8a9d531a6be/scijava/scijava-ops-engine/pom.xml#L113) 😁